geoh5py/ui_json/input_file.py

```diff
@@ -274,7 +274,10 @@ def update_ui_values(self, data: dict):
                 if (value is None) and (not self.ui_json[key].get("enabled", False)):
                     continue
 
-                self.ui_json[key][member] = value
+                if isinstance(value, dict):
```

Comment:
See note below.  This function expects a flat dictionary, so there shouldn't be any need to handle a dictionary type here.

Reply:
Normally, the "value" is not a dictionary. 
It's only a dictionary for the range slider widget.

Reply:
It's a bit more complicated. 

The widgets were returning only one object: in "value"

But with more complex widgets, it returns more than one value...